### DISCUSSION TITLE
V1.6.4 Unable to initialize Policy Plugin

### DISCUSSION THREAD

**Discussion by PA7R14RCH** [containers/podman#9787] [Q&A]
Using podman 1.6.4 on Rhel 7.9 and running into this issue of not being able to "sudo su" inside of a container and getting the following results:

sudo: unable to change to root gid: Operation not permitted
sudo: unable to initialize policy plugin

Found this issue on both a VM and bare metal Server.

Is there a fix for this I can apply to a previous version? Current environment does not allow for a version update. I've been up and down this container with little success.

**Comment by mheon** [comment]
I think this is fully expected unless you're running the container as privileged - the container is restricted to prevent it from gaining new privileges (for security reasons) by default.

**Reply by mheon** [reply]
Hm. From https://github.com/containers/podman/issues/6645 I may be wrong here.

If you're willing to apply patches, I strongly suggest you just upgrade to a more recent version. The v1.6 branch is ancient by this point, and the code has diverged *majorly* - no backport there is going to be easy.

**Comment by rhatdan** [comment]
We will not be updating podman on RHEL7 any longer unless there is a major CVE and then just to fix the security issue.

Can you upgrade to RHEL8?
